scripts/target-metadata.pl

```diff
@@ -27,6 +27,7 @@ (@)
 		/ext4/ and $ret .= "\tselect USES_EXT4\n";
 		/targz/ and $ret .= "\tselect USES_TARGZ\n";
 		/cpiogz/ and $ret .= "\tselect USES_CPIOGZ\n";
+		/kernel2mikrotikyaffs2/ and $ret .= "\tselect USES_KERNEL2MIKROTIKYAFFS2\n";
```

Comment:
Thats quite a bulky name, maybe we should simply call it `USES_MINOR` and `minor`?